home.js

```diff
@@ -10,4 +10,195 @@ window.onload = function () {
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Profesional Art',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Data Dashboard',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+function showPopup() {
+  document.querySelector('.multi-cont').style.display = 'flex';
+}
+
+/* ----------------- Building Popup (Mobile) ---------------- */
+document.addEventListener('DOMContentLoaded', (event) => {
+  const popupSection = document.querySelector('.multi-cont');
+  const popupCont = document.createElement('div');
+  popupCont.className = 'multi-cont2';
+  popupSection.append(popupCont);
+  const closeSign = document.createElement('img');
+  closeSign.classList.add('closesign');
+  closeSign.src = './assets/Icon.png';
+  closeSign.alt = 'closesign';
+  closeSign.onclick = hidePopup;
+  const popupTitle = document.createElement('h3');
+  popupTitle.className = 'multi-stories';
+  const popupUl = document.createElement('ul');
+  popupUl.className = 'ul-tech1';
+  const popupImg = document.createElement('img');
+  popupImg.alt = 'project';
+  popupImg.className = 'image1';
+  const popupDescription = document.createElement('p');
+  popupDescription.className = 'multitext';
+  const liveButton = document.createElement('button');
+  liveButton.innerText = 'See Live';
+  liveButton.className = 'btn-pop';
+  const sourceButton = document.createElement('button');
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
```

Comment:
- [ ] For the desktop screens, The pop-up doesn't match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) design and I can't see the project details. Please make sure that the project details are visible on the desktop screens and design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) template.


### Your design
![image](https://user-images.githubusercontent.com/83803414/202738191-35d471de-792a-43c8-9153-b7d515b5489f.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/202738241-87e24f7b-b010-403f-83d5-eae2a7571e97.png)